Whiskey/Tasks/New-WhiskeyProGetUniversalPackage.ps1

```diff
@@ -84,27 +84,29 @@ function New-WhiskeyProGetUniversalPackage
         $parentPathParam['ParentPath'] = $sourceRoot
     }
 
-    if( -not $manifestProperties.ContainsKey('Name') )
+    if( -not $manifestProperties.ContainsKey('name') )
     {
-        $manifestProperties['Name'] = $name
+        $manifestProperties['name'] = $name
     }
 
-    if( -not $manifestProperties.ContainsKey('Title') )
+    if( -not $manifestProperties.ContainsKey('title') )
     {
-        $manifestProperties['Title'] = $name
+        $manifestProperties['title'] = $name
     }
 
-    if( -not $manifestProperties.ContainsKey('Description') )
+    if( -not $manifestProperties.ContainsKey('description') )
     {
-        $manifestProperties['Description'] = $TaskParameter['Description']
+        $manifestProperties['description'] = $TaskParameter['Description']
     }
 
-    if( -not $manifestProperties.ContainsKey('Version') )
+    if( -not $manifestProperties.ContainsKey('version') )
     {
-        $manifestProperties['Version'] = $version.SemVer2NoBuildMetadata.ToString()
+        $manifestProperties['version'] = $version.SemVer2NoBuildMetadata.ToString()
     }
 
-    $tempRoot = $TaskContext.Temp
+    $tempRoot = Join-Path -Path $TaskContext.Temp -ChildPath ('Temp.{0}.upack' -f $name)

```

Comment:
I'm worried about paths that are too long. Maybe call the directory "upack"? Temp directory's are just for each task so there wouldn't be any collisions.

Reply:
Oh good point, fixed.